client/src/components/Navbar/Navbar.tsx

```diff
@@ -113,8 +113,11 @@ const Navbar: React.FC = () => {
 
   const renderMenuItems = () => {
     // TODO: conditionally render based on profile type
+    //   Profiles have an attribute accountType
+    //   NEED_TO: Make a request to get the associated profile
+    const profile = { accountType: 'pet_owner' }; //dummyProfile
     return menuItems.map((menu) => {
-      if (menu.authenticated) {
+      if (menu.authenticated && menu.canView?.includes(profile.accountType)) {
```

Comment:
Please check the type of the logged in user here (as opposed to using mock data) 😄

Reply:
Hey i might be misunderstanding the scope of the ticket, but as of now we do not have a route to the backend to provide an associated profile. Should I be creating that backend route and setting it up on the frontend for this as well?

Reply:
I missed the fact that you could grab the profile from the auth context. I also updated the backend to give you the profile data on login as well (not sure if this was redundant).

Reply:
@natey37 It would be ideal if we could fetch the logged in user's profile and check the account type. I noticed that you've added the profile as a response to one of the auth controllers, good thinking! However, it would make more sense if we have a seperate route to fetch the associated user's profile